src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -31,62 +28,61 @@ public class TypicalPersons {
             .build();
     public static final Person FIONA = new PersonBuilder().withName("Fiona Kunz").withPhone("94824271")
             .withEmail("fiona@example.com").withTelegram("fionaTelegram")
-            .withAttendance("0,0,0,0,0,0,0,0,0,0,0,0", "0,0,0,0,0,0,0,0,0,0,0,0")
+            .withAttendance(Attendance.ORIGINAL_ATD, Attendance.ORIGINAL_PART)
             .build();
     public static final Person GEORGE = new PersonBuilder().withName("George Best").withPhone("94824424")
             .withEmail("george@example.com").withTelegram("georgeTelegram")
-            .withAttendance("0,0,0,0,0,0,0,0,0,0,0,0", "0,0,0,0,0,0,0,0,0,0,0,0").build();
+            .withAttendance(Attendance.ORIGINAL_ATD, Attendance.ORIGINAL_PART).build();
 
     // Manually added
-    public static final Person HOON = new PersonBuilder().withName("Hoon Meier").withPhone("84824249")
-            .withEmail("hoon@example.com").withTelegram("hoonTelegram")
-            .withAttendance("0,0,0,0,0,0,0,0,0,0,0,0", "0,0,0,0,0,0,0,0,0,0,0,0").build();
-    public static final Person IDA = new PersonBuilder().withName("Ida Mueller").withPhone("84821318")
-            .withEmail("ida@example.com").withTelegram("idaTelegram")
-            .withAttendance("0,0,0,0,0,0,0,0,0,0,0,0", "0,0,0,0,0,0,0,0,0,0,0,0").build();
+    public static final Person HOON = new PersonBuilder().withName("Hoon Meier").withPhone("8482424")
+            .withEmail("stefan@example.com").withTelegram("littleindia")
+            .withAttendance(Attendance.ORIGINAL_ATD, Attendance.ORIGINAL_PART).build();
+    public static final Person IDA = new PersonBuilder().withName("Ida Mueller").withPhone("8482131")
+            .withEmail("hans@example.com").withTelegram("chicagoave")
+            .withAttendance(Attendance.ORIGINAL_ATD, Attendance.ORIGINAL_PART).build();
 
     // Manually added - Person's details found in {@code CommandTestUtil}
     public static final Person AMY = new PersonBuilder().withName(VALID_NAME_AMY).withPhone(VALID_PHONE_AMY)
             .withEmail(VALID_EMAIL_AMY).withTelegram(VALID_TELEGRAM_AMY).withTags(VALID_TAG_FRIEND)
-            .withAttendance(VALID_ATTENDANCE_AMY, VALID_PART_AMY).build();
+            .withAttendance(Attendance.ORIGINAL_ATD, Attendance.ORIGINAL_PART).build();
     public static final Person BOB = new PersonBuilder().withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
             .withEmail(VALID_EMAIL_BOB).withTelegram(VALID_TELEGRAM_BOB).withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND)
-            .withAttendance(VALID_ATTENDANCE_BOB, VALID_PART_BOB).build();
+            .withAttendance(Attendance.ORIGINAL_ATD, Attendance.ORIGINAL_PART).build();
 
 
-    public static final String KEYWORD_MATCHING_MEIER = "Meier";
+    public static final String KEYWORD_MATCHING_MEIER = "Meier"; // A keyword that matches MEIER
 
     private TypicalPersons() {
-        // prevents instantiation
-    }
+    } // prevents instantiation
```

Comment:
Is there a reason for preferring this comment style in the constructor? A comment after a closing bracket `}` feels unintuitive.